x/payment/types/payfordata_test.go

```diff
@@ -419,7 +419,7 @@ func validWirePayForData(t *testing.T) *MsgWirePayForData {
 	msg, err := NewWirePayForData(
 		[]byte{1, 2, 3, 4, 5, 6, 7, 8},
 		bytes.Repeat([]byte{1}, 2000),
-		16, 32, 64,
+		4, 8, 16, 32, 64, 128,
```

Comment:
[for reviewers]
Updated this test data generator because a message with 2000 bytes expects all these square sizes. This intuitively makes sense b/c IIUC the algorithm that defines which square sizes must be signed over starts at max (128) and proceeds by lower powers of two until a square size is lower than the message size.

In this case 2000 bytes can't fit in a square size of 2 x 2 x 256 = 1,024 bytes so 4 is the lowest square needed.